exercises/D-logical-operators/exercise.js

```diff
@@ -11,29 +11,56 @@ var cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-var htmlLevelAbove5;
-var cssLevelAbove5;
+
+
+function htmlLevelAbove5() {
+    if (htmlLevel > 5) {
+        return true;
+    }
+    return false;
+}
+
+function cssLevelAbove5() {
+    if (cssLevel > 5) {
+        return true;
+    }
+    return false;
+}
+
+
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-var cssAndHtmlAbove5;
-var cssOrHtmlAbove5;
+function cssAndHtmlAbove5() {
+    if (htmlLevel > 5 && cssLevel > 5) {
+        return true;
+    }
+    return false;
+}
+
+function cssOrHtmlAbove5() {
+    if (htmlLevel > 5 || cssLevel > 5) {
+        return true;
+    }
+    return false;
+}
 
-/* 
-  DO NOT EDIT BELOW THIS LINE
-  --------------------------- */
 
-console.log("Is Html knowledge above 5?", htmlLevelAbove5);
```

Comment:
If you do it like the comment above, it should work with these expected results 🙂